test-suite/src/test/java/io/micronaut/kafka/docs/consumer/offsets/ack/ProductListener.java

```diff
@@ -1,24 +1,23 @@
-package io.micronaut.configuration.kafka.docs.consumer.offsets.ack;
+package io.micronaut.kafka.docs.consumer.offsets.ack;
 
 import io.micronaut.configuration.kafka.annotation.KafkaListener;
 import io.micronaut.configuration.kafka.annotation.Topic;
-import io.micronaut.configuration.kafka.docs.consumer.config.Product;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.kafka.docs.Product;
 import io.micronaut.messaging.Acknowledgement;
 
 import static io.micronaut.configuration.kafka.annotation.OffsetReset.EARLIEST;
 import static io.micronaut.configuration.kafka.annotation.OffsetStrategy.DISABLED;
 
-class ProductListener {
+@Requires(property = "spec.name", value = "AckProductListenerTest")
+// tag::clazz[]
+@KafkaListener(offsetReset = EARLIEST, offsetStrategy = DISABLED) // <1>
```

Comment:
why did you move the `@KafkaListener` to the class level ?

Reply:
@sdelamo 

> why did you move the @KafkaListener to the class level ?

We need to show that line in the example, because it's linked to bullet point `<1> Committing offsets automatically is disabled`. So it does make sense to me.